### ISSUE TITLE
Download database function broken

### ISSUE THREAD

**Issue by mliu49** [ReactionMechanismGenerator/RMG-website#153]
```
Internal Server Error: /database/export_zip Traceback (most recent call last):
  File "/opt/anaconda/envs/rmg_env/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/RMG-website/rmgweb/database/views.py", line 122, in export
    subprocess.check_call(cmd_export)
  File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
CalledProcessError: Command '['python', '/var/www/RMG-database/input../scripts/exportOldDatabase.py', '/var/www/RMG-website/rmgweb/../database/export/RMG_database']' returned non-zero exit status 2
```

**Comment by mliu49** [comment]
See ReactionMechanismGenerator/RMG-Py#1021. Planning to replace with a GitHub download link.

**Comment by mliu49** [comment]
Implemented in dc7ecf0637ae4e4cf506dceea86fb74715a8981b, changes are live.
